packages/libpluto/build.sh

```diff
@@ -0,0 +1,36 @@
+TERMUX_PKG_HOMEPAGE=https://plutolang.github.io/
+TERMUX_PKG_DESCRIPTION="Shared library for the Pluto interpreter"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="Komo @cattokomo"
+TERMUX_PKG_VERSION="0.9.4"
+TERMUX_PKG_REVISION=2
```

Comment:
This is a new package, remove the revision.
What you probably want instead is a BREAKS/REPLACES.
```suggestion
TERMUX_SUBPKG_BREAKS="plutolang (<< 0.9.4-2)"
TERMUX_SUBPKG_REPLACES="plutolang (<< 0.9.4-2)"
```